7-bank-project/README.md

```diff
@@ -16,6 +16,6 @@ In this project, you'll learn how to build a fictional bank. These lessons inclu
 
 These lessons were written with :hearts: by [Yohan Lasorsa](https://twitter.com/sinedied).
 
-If you're interested to learn how to build the [server API](./api/README) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
+If you're interested to learn how to build the server API used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
```

Comment:
Is the link broken? Can we fix it rather than remove it?

Reply:
Good idea!  Today's my first day of contribution. Please let me know if I am missing anything. :)